nasa-gibs/worldview
# WV-3103: Add 4 Sentinel 3A and 3B orbit track layers

## Description

Fixes #WV-3103.

Add 4 Sentinel 3A and 3B orbit track layers - "OrbitTracks_Sentinel-3A_Ascending", "OrbitTracks_Sentinel-3A_Descending", "OrbitTracks_Sentinel-3B_Ascending", "OrbitTracks_Sentinel-3B_Descending"

## How To Test
1. `git checkout wv-3103-sentinel3a-3b-orbit-tracks`
2. `npm run build && npm start`
3. Open with [these URL parameters](http://localhost:3000/?v=-129.55717830915978,-47.56998519789069,61.688312802665635,40.743377713433944&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,OrbitTracks_Sentinel-3B_Descending,OrbitTracks_Sentinel-3B_Ascending,OrbitTracks_Sentinel-3A_Descending,OrbitTracks_Sentinel-3A_Ascending,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-04-28-T15%3A05%3A16Z)
4. Check to make sure orbit track layers display at global scales, and zoomed in scales. When you zoom in, check to make sure you can click on the points on the tracks to get metadata. Check layers work in all projections. Ensure you can take image snapshots of the layers.

@nasa-gibs/worldview
